framework_tvb/tvb/adapters/visualizers/surface_view.py

```diff
@@ -129,6 +129,10 @@ def get_url_for_region_boundaries(surface_gid, region_mapping_gid, adapter_id):
         return URLGenerator.build_url(adapter_id, 'generate_region_boundaries', surface_gid,
                                       parameter='region_mapping_gid=' + region_mapping_gid)
 
+    @staticmethod
+    def get_url_for_data_page_split(surface_gid, adapter_id):
```

Comment:
I suggest drop this new method, and use directly `URLGenerator .build_url` in `brain.py`